Makefile

```diff
@@ -23,12 +23,14 @@ lint:
 	pipenv check
 	pipenv run isort .
 	pipenv run black --line-length 120 .
+	pipenv run djlint .
 	pipenv run flake8
 
 lint-check: load-design-system-templates
 	pipenv check
 	pipenv run isort . --check-only
 	pipenv run black --line-length 120 --check .
+	pipenv run djlint .
```

Comment:
I think you need to add the `--check` flag here right? so that it's `pipenv run djlint . --check`.  Otherwise I think it'll format it for you and then carry on, instead of failing the build which is what we want

Reply:
`--check` seems to be used as the formatter in DJlint
<img width="1121" alt="Screen Shot 2022-10-11 at 15 05 27 pm" src="https://user-images.githubusercontent.com/113037557/195113351-0d01f53d-28f2-4ffe-9853-4a79534f663b.png">
When run the build fails as the code needs to be reformatted to match the suggested output (which does not look good). 
When run without `--check` it only checks the linting errors and fails if those are present. Is the formatter required?

Reply:
yeah sorry ^^ I missed that check was only for the formatter.  Also linters by definition check themselves >_<